utils/gyb_syntax_support/StmtNodes.py

```diff
@@ -197,8 +197,8 @@
              Child('Condition', kind='Syntax',
                    node_choices=[
                        Child('Expression', kind='Expr'),
-                       Child('Availablity', kind='AvailabilityCondition'),
-                       Child('Unavailablity', kind='UnavailabilityCondition'),
+                       Child('Availability', kind='AvailabilityCondition'),
+                       Child('Unavailability', kind='UnavailabilityCondition'),
```

Comment:
I think these changes may be causing some skew in generated sources that need to match the SwiftSyntax repository:

```
diff --recursive --exclude '.*' --context=0 /var/folders/pf/p5g2vb2j0yx1x6vlmxdppc700000gn/T/tmpx7cx6_tm/SyntaxBuilders.swift /Users/ec2-user/jenkins/workspace/swift-PR-macos/branch-main/swift-syntax/Sources/SwiftSyntax/gyb_generated/SyntaxBuilders.swift
*** /var/folders/pf/p5g2vb2j0yx1x6vlmxdppc700000gn/T/tmpx7cx6_tm/SyntaxBuilders.swift	Thu Apr 21 21:39:42 2022
--- /Users/ec2-user/jenkins/workspace/swift-PR-macos/branch-main/swift-syntax/Sources/SwiftSyntax/gyb_generated/SyntaxBuilders.swift	Thu Apr 21 15:19:40 2022
***************
*** 6434,6435 ****
!   public mutating func useDeclname(_ node: DeclNameSyntax) {
!     let idx = TargetFunctionEntrySyntax.Cursor.declname.rawValue
--- 6434,6435 ----
!   public mutating func useDelcname(_ node: DeclNameSyntax) {
!     let idx = TargetFunctionEntrySyntax.Cursor.delcname.rawValue
```

The easiest path to unblocking the rest of the changes would be to revert the changes in this file, but we could also make a corresponding PR in https://github.com/apple/swift-syntax

Reply:
Most of the other changes to `gyb_syntax_support` require corresponding changes to https://github.com/apple/swift-syntax. Could you create a matching PR in that repo by 
- Checking out swift-syntax next to swift
- Run `swift-syntax/build-script.py --toolchain doesnt-matter --degyb-only`

Alternatively, we could consider dropping all changes in `gyb_syntax_support` from this PR.

Reply:
I've dropped them from this PR.

Reply:
👍 If you want to open a separate PR for the `gyb_syntax_support` changes, I would really appreciate it. If you need help with the corresponding swift-syntax change, I’m very happy to help you out!